src/org/omegat/gui/editor/EditorController.java

```diff
@@ -1456,6 +1456,36 @@ public void nextTranslatedEntry() {
         nextTranslatedEntry(true);
     }
 
+    private void linkedEntry(boolean forward, String linked) {
+        iterateToEntry(forward, ste -> {
+                TMXEntry info = Core.getProject().getTranslationInfo(ste);
+                if (String.valueOf(info.linked).equals(linked)) {
+                    return true;
+                }
+                return false;
+            });
```

Comment:
This is one of the things that makes programming so interesting.
My solution for line 1462 to 1464 was:

```java
if (linked.equals(String.valueOf(info.linked))) {
    return true;
}
```
It worked on a brief test, and passed the SpotBugs check mentioned by Hiroshi. I'll have to study his solution to learn why it's a better idiom.

Reply:
the direction of that comparison isn't too important, I guess, we need to make sure
that `info.linked` and `linked` are identical

Reply:
Hiroshi's is better because he uses one `return` statement which work for both cases